common/culture/traditions/wc_societal_traditions.txt

```diff
@@ -0,0 +1,374 @@
+﻿################################
+# SOCIETAL CULTURAL TRADITIONS #
+################################
+
+tradition_arkonite = {
+	category = societal
+
+	layers = {
+		0 = learning
+		1 = indian
+		4 = arkonite.dds
+	}
+	
+	can_pick = {
+		
+	}
+	
+	parameters = {
+		prowess_from_religious_traits = yes
+		culture_clergy_can_fight = yes
+	}
+	character_modifier = {
+		caster_damage_mult = 0.1
+		caster_toughness_mult = 0.05
+		knight_effectiveness_mult = 0.05
+		development_growth_factor = 0.2
+	}
+	
+	cost = {
+		prestige = {
+			add = {
+				value = tradition_base_cost
+				desc = BASE
+				format = "BASE_VALUE_FORMAT"
+			}
+			if = {
+				limit = {
+					NOR = {
+						any_in_list = { list = traits this = flag:ethos_bellicose }
+						any_in_list = { list = traits this = flag:ethos_spiritual }
+					}
+				}
+				add = {
+					value = tradition_incompatible_ethos_penalty
+					desc = not_bellicose_or_spiritual_desc
+				}
+			}
+			if = {
+				limit = {
+					trigger_if = {
+						limit = {
+							scope:character = {
+								is_ai = no
+							}
+						}
+						NOT = {
+							scope:character = {
+								save_temporary_scope_as = culture_head_link
+								faith = {
+									any_faith_character = {
+										count >= 15
+										culture = scope:culture_head_link.culture
+										prowess >= 10
+										OR = {
+											is_clergy = yes
+											has_education_learning_trigger = yes
+										}
+									}
+								}
+							}
+						}
+					}
+					trigger_else = {
+						always = no
+					}
+				}
+				add = {
+					value = tradition_unfulfilled_criteria_penalty
+					desc = culture_head_faith_clergy_prowess_desc
+				}
+			}
+			
+			multiply = tradition_replacement_cost_if_relevant
+		}
+	}
+	
+	ai_will_do = {
+		value = 100
+		if = {
+			limit = {
+				NOR = {
+					any_in_list = { list = traits this = flag:ethos_bellicose }
+					any_in_list = { list = traits this = flag:ethos_spiritual }
+				}
+			}
+			multiply = 0.25
+		}
+		# Warcraft
+		if = {
+			limit = {
+				scope:character.faith = {
+					has_doctrine = doctrine_pluralism_fundamentalist
+				}
+			}
+			multiply = 1.15
+		}
+	}
+}
+
+tradition_bandit_culture = {
```

Comment:
In the base game, post feudal raiding is culturally permitted only by `tradition_practiced_pirates`. I have assigned this tradition to the following cultures: 
``` 
#Seafaring cultures 
Northsea 
Blackwater, Gilblin 
Kvaldir 
Pirate Culture, Wastewander 
``` 
Jungle, Forest, Mountain Hozen, and Saurok are already assigned to `tradition_practiced_pirates` but are intended to be replaced by `tradition_might_is_right` (Added in 09654e783 ). Other cultures which may potentially start with `tradition_might_is_right` include Warsong, Gnoll, Quilboar, Ogre, and Centaur, subject to further discussion. Groups with deathbringer, doombringer and shathgral faiths are discounted as they have `doctrine_end_justifies_means`, which also permits raiding.

The idea behind `tradition_might_is_right` is to allow combative/unruly societies to continue menacing their neighbours even after their feudalise. WarCraft is about war, and feudalising in the course of a playthrough should not reduce the overall level of conflict. Vanilla offers `tradition_quarrelsome` as a solution, however, it fails to encapsulate the greyness of conflict since it simply lowers the threshold for de jure wars. A raiding tradition allows cultures where leadership is often contested and loose to simulate the contestation of borderlands during purported peacetime (e.g. Ashenvale skirmishing), opportunism by neutral neighbours during large conflicts and the limited ability of rulers of unruly cultures to completely control their vassals.

Because gaining the ability to raid is so useful. It may outshine the Vanilla tradition `tradition_warrior_culture` Thus, I have designed certain drawbacks to the tradition, such as 
```
monthly_county_control_change_factor = -0.2
opinion_of_liege = -10
cultural_head_fascination_mult = -0.1
```
to encourage on players who are set on this playstyle to select this culture. `tradition_practiced_pirates` remains a purely loot-oriented tradition due to `max_loot_mult = 0.25` and it is more flexible (for land and sea) despite having no buffs. This balancing is subject to feedback @zumbak04 

@Gaben1025 From what I see in your code, `tradition_bandit_culture` is meant to be a wc replacement for `tradition_warrior_culture` in specific cases. Can you share which cultures you had in mind?

Reply:
Per our conversation, `tradition_might_is_right` will act as the WC cultural feudal raiding tradition, while `tradition_bandit_culture` will be focused as a WC culture specific upgrade to `tradition_swords_for_hire`.
Possible modifiers include but are not limited to:
```
cultural opinion
loot
guest, courtier visitation and recruitment
hiring
reinforcement and supply duration
```
Can mark as closed and/or discuss further ideas at your discretion.